scripts/README.md

```diff
@@ -4,6 +4,19 @@
 
 A typical release goes like this:
 
+### Prerequisites
+
+1. You must be a member of the `@mui` org in npm to publish the release
+1. Set up your npm authToken by logging into npm (`npm login`) . This will save a token to `~/.npmrc` as a line that looks
```

Comment:
Could you update the numbering? It looks better when reading plain text (not rendered markdown). I wouldn't mind having correct indexes in other lists in this file as well.